patches/api/0429-Add-Entity-Body-Yaw-Api.patch

```diff
@@ -0,0 +1,30 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TheTuso <piotrekpasztor@gmail.com>
+Date: Fri, 27 Jan 2023 22:18:33 +0100
+Subject: [PATCH] Add Entity Body Yaw Api
+
+
+diff --git a/src/main/java/org/bukkit/entity/LivingEntity.java b/src/main/java/org/bukkit/entity/LivingEntity.java
+index ead90b002034c463983af948a4fb882109b8d3a2..827401307bd219331ca71e3836880d1e11a37252 100644
+--- a/src/main/java/org/bukkit/entity/LivingEntity.java
++++ b/src/main/java/org/bukkit/entity/LivingEntity.java
+@@ -1106,5 +1106,19 @@ public interface LivingEntity extends Attributable, Damageable, ProjectileSource
+      * @param amount the amount of damage to do
+      */
+     void damageItemStack(org.bukkit.inventory.@NotNull EquipmentSlot slot, int amount);
++
++    /**
++     * Gets entity body yaw
++     *
++     * @return entity body yaw
++     */
++    float getBodyYaw();
```

Comment:
The restrictions a yaw value follows are rather loosely defined (or non existent).
https://jd.papermc.io/paper/1.19/org/bukkit/Location.html#getYaw() already partially lines out the values yaw values represent.

I think this method would benefit from a `@see` reference to the location#getYaw method as well as definitions on the expected value range.